Misc/NEWS.d/next/Library/2024-06-05-11-03-10.gh-issue-120029.QBsw47.rst

```diff
@@ -1,4 +1,4 @@
-Expose :meth:`symtable.Symbol.is_comp_iter` and :meth:`symtable.Symbol.is_comp_cell`
-in the :mod:`symtable` module.
+Expose :meth:`symtable.Symbol.is_free_class`, :meth:`symtable.Symbol.is_comp_iter`
```

Comment:
A tip:  the resulting text will look better if write it as
```rst
Expose :class:`symtable.Symbol` methods :meth:`~symtable.Symbol.is_free_class`, :meth:`~symtable.Symbol.is_comp_iter`, ...
```
and remove ``` in the :mod:`symtable` module ``` because the module name is already included in the class reference.

Or keep ``` in the :mod:`symtable` module ```, but hide the module name in the class reference: ``` :class:`~symtable.Symbol` ```.

It does not matter much in the NEWS entry, but it would be more important in the What's New document. BTW, you can also add this in the What's New document.